bridge_adaptivity/module/models.py

```diff
@@ -232,6 +248,10 @@ def engine_driver(self):
             self.DRIVER = engine_driver
         return self.DRIVER
 
+    @property
```

Comment:
as for me it looks like it should return dict, but it return tuple.

should it be better to rename this function to make it more clear?

Reply:
for example 
```
def lti_params_list(...):
    ....
```
?

Reply:
I don't want to have too long property name, btw property returns generator not list and it could be confusing expect list and get a generator. lti_params looks good to me.